tfx/components/statistics_gen/executor_test.py

```diff
@@ -164,8 +171,49 @@ def testDoWithTwoSchemas(self):
     # Run executor.
     stats_gen_executor = executor.Executor()
     with self.assertRaises(ValueError):
-      stats_gen_executor.Do(
-          input_dict, output_dict, exec_properties=exec_properties)
+      stats_gen_executor.Do(input_dict, output_dict, exec_properties)
+
+  def testDoWithExcludeSplits(self):
```

Comment:
consider merge the test (you can create a fake split, and exclude that, so you test both `exclude` functionality and multiple splits output functionality)